package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "ses",
-  "version": "0.5.0",
+  "version": "0.5.1",
```

Comment:
I'd like to hear about best-practices for version-string management, but I'm inclined to think a PR shouldn't make it look like we've just released a new version.. maybe we should land the PR, then only change `version` when we spin the next release (which will probably be right after we land it, but still a separate step).

Reply:
Done, with "0.5.0+dev".